tests/test_checker/test_noqa.py

```diff
@@ -185,6 +185,7 @@
     'WPS447': 1,
     'WPS448': 1,
     'WPS449': 1,
+    'WPS450': 0,
```

Comment:
This violation should be present in `tests/fixtures/noqa.py`
And this should equal to `1`